packages/api/internal/orchestrator/node.go

```diff
@@ -139,12 +149,25 @@ func (o *Orchestrator) GetNode(nodeID string) *Node {
 	return n
 }
 
+// clusterNodeID - this way we don't need to worry about multiple clusters with the same node ID in shared pool
+func (o *Orchestrator) clusterNodeID(clusterID uuid.UUID, nodeID string) string {
+	clusterPrefix := clusterID.String()[0:7]
+	return fmt.Sprintf("%s-%s", clusterPrefix, nodeID)
+}
```

Comment:
Thinking if there is a better way. We want to prefix cluster nodes so there is no collision where multiple clusters have the same node ID.

This takes the first 8 chars from the cluster UUID (generated by us) and puts it before the node ID.

Reply:
Resolved, NodeID is no longer returned to the user, because we stopped using the client part in sandbox, so it can be longer and contain characters that are not sandbox ID friendly, such as dashes.